src/vmm/src/vstate/memory.rs

```diff
@@ -694,7 +649,105 @@ mod tests {
     }
 
     #[test]
-    fn test_mark_dirty_mem() {
+    fn test_from_raw_regions_unguarded() {
+        // Check dirty page tracking is off.
```

Comment:
can be 2 tests

Reply:
I don't think splitting this is worth it. This test is small by itself, and it is almost same as `test_from_raw_regions` test.